src/podofo/private/FreetypePrivate.cpp

```diff
@@ -87,12 +87,12 @@ FT_Face FT::CreateFaceFromBuffer(const bufferview& view, unsigned faceIndex,
             buffer = view;
 
         // Unconditionally re-create the face from the copied buffer
-        return createFaceFromBuffer(buffer, 0);
+        return createFaceFromBuffer(view, 0);
     }
     else
     {
         buffer = view;
-        return createFaceFromBuffer(buffer, 0);
+        return createFaceFromBuffer(view, 0);
     }
```

Comment:
This change doesn't look like it changes anything. After line 94, the two variables should have the same data anyway.

Reply:
I have descibed the [issue ](https://github.com/podofo/podofo/issues/246). Let me know please if I didn't described it clear. It took also some time to figure out why it happens. But basically buffer in this case is local variable which destoys after font is created and `FT_Face` has reference to destoyed buffer.